packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -80,12 +80,59 @@ export 'package:flutter/services.dart' show
 
 // BIDIRECTIONAL TEXT SUPPORT
 
+/// An [InheritedElement] that has hundreds of dependencies but will
```

Comment:
nit: a future reader will probably appreciate a little more documentation on how this is different from the regular InheritedElement.

Reply:
Done.